hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.js

```diff
@@ -38,40 +38,54 @@ const createQuantityList = (number) => {
 };
 
 export const AddToCartForm = ({
-    availableQuantity,
+    quantityAvailable,
+    constraintMax,
     handleSubmit,
     handleChange,
     requestFailure,
     values: { quantity },
 }) => {
+    let dropdownNum;
+    if (!constraintMax) {
+        dropdownNum = quantityAvailable;
+    } else {
+        dropdownNum = Math.min(quantityAvailable, constraintMax);
+    }
+
     return (
         <>
             {requestFailure && (
-                <Alert className={styles.alert} variant="filled" severity="error">
+                <Alert variant="filled" severity="error">
                     {requestFailure.message}
                 </Alert>
             )}
             <form className={styles.form} onSubmit={handleSubmit}>
                 <FormControl variant="outlined" className={styles.formControl}>
                     <InputLabel>Qty</InputLabel>
```

Comment:
In order to be used properly with `getByLabelText` (and for accessibility reasons), this should have an `id`